community-content/pipeline_components/PyTorch/Train_PyTorch_model/from_CSV/component.yaml

```diff
@@ -0,0 +1,209 @@
+name: Train pytorch model from csv
+description: Trains PyTorch model
+metadata:
+  annotations:
+    author: Alexey Volkov <alexey.volkov@ark-kun.com>
+    canonical_location: 'https://raw.githubusercontent.com/Ark-kun/pipeline_components/master/components/PyTorch/Train_PyTorch_model/from_CSV/component.yaml'
```

Comment:
Are these locations up to date?

Reply:
>Are these locations up to date?
Yes. These locations have the latest version of components. (And many other accompanying components)
However the pipelines always link to specific immutable component versions.